scaffolding-chef-inspec/lib/run.ps1

```diff
@@ -0,0 +1,59 @@
+$env:SSL_CERT_FILE="{{pkgPathFor "scaffold_cacerts"}}/ssl/cert.pem"
+$env:SSL_CERT_DIR="{{pkgPathFor "scaffold_cacerts"}}/ssl/certs"
+$env:PATH = "{{pkgPathFor "scaffold_inspec_client"}}/bin;$env:PATH"
```

Comment:
this line can get removed if the fix is adding the `pkgPathFor` to the inspec call.